sources/packages/backend/apps/api/src/route-controllers/student-appeal/student-appeal.students.controller.ts

```diff
@@ -111,40 +104,11 @@ export class StudentAppealStudentsController extends BaseController {
         "There is already a pending appeal for this student.",
       );
     }
-    let dryRunSubmissionResults: DryRunSubmissionResult[] = [];
-    try {
-      const dryRunPromise = payload.studentAppealRequests.map((appeal) =>
```

Comment:
Please refer to the comment https://github.com/bcgov/SIMS/pull/1681/files#r1087213717

Student appeal can still use dry run.